trade-finance-manager-api/src/v1/api.js

```diff
@@ -622,6 +621,21 @@ const addUnderwriterCommentToGefDeal = async (dealId, commentType, comment) => {
   }
 };
 
+const getAllFacilities = async () => {
+  try {
+    const response = await axios({
+      method: 'GET',
+      url: `${centralApiUrl}/v1/tfm/facilities`,
```

Comment:
Is it missing id from `/tfm/facilities/:id`?

Reply:
this is a new endpoint listing all facilities (both from GEF and BSS) from TFM. we don't need the `:id` in this case